### ISSUE TITLE
Accessibility Concern with new Asset Page Background

### ISSUE THREAD

**Issue by dcashpeterson** [pnp/custom-learning-office-365#771]
I saw today's post and notification about the gray background update, but I can't find any information about why the gray background was added.  This background color actually makes the assets very difficult to read due to the poor contrast, and it is going to cause issues for individuals with vision challenges.  Can the gray background be removed or changed to a _much_ lighter color?

![LP gray background](https://github.com/pnp/custom-learning-office-365/assets/123119748/2b040b3e-8704-41cd-bb9f-113165bfd1ee)

_Originally posted by @renwein in https://github.com/pnp/custom-learning-office-365/issues/767#issuecomment-1919968211_

**Comment by clarelcampbell** [comment]
Hi - would like to add that we have concern about this as well so hope that a) the text can be made black b) the background can be lightened, thanks

**Comment by dcashpeterson** [comment]
@clarelcampbell Accessibility is 100% on our radar screen for Microsoft Learning Pathways. I checked the contrast ratio of the pages and they currently meet WCAG AA contrast ratio standards for all text, form elements, and headers. It does not meet WCAG AAA standards. I am meeting with the team to determine what we can do about this. I will keep this issue updated with progress.

**Comment by jhwang-ms** [comment]
Hello! We are aware that trying to fix one thing created more issues and are trying to work through those right now. Unfortunately, there are more hoops for us to jump through than we anticipated but we're trying to get to a solution as fast as we can. I hope to get a fix in place this month.

**Comment by TomMeadows1990** [comment]
Hi there, I wondered if there was an update on this? The length of the grey box continues long after the end of the content (see attachment). It's a greta feature which I use a lot, but this issue means I can't use the feature without the webpages looking amateurish. Any update would be appreciated. Many thanks, Tom
![2024-04-12_16-27-50](https://github.com/pnp/custom-learning-office-365/assets/166843926/ab6fef47-d493-4600-beca-d416572b00df)

**Comment by dcashpeterson** [comment]
@TomMeadows1990  the page height isn't an accessibility concern so it will not be tackled in this issue. We are working on addressing some technical challenges with the content see #786 for more details as they progress.

**Comment by dcashpeterson** [comment]
Hi folks, It's been a while on this issue. We have finally gotten the page background back to white for most of the assets which is what this issue was about. I am going through the assets one by one now and making sure they work and documenting the content that doesn't work. Please follow #786 for updates on that and the content updates.
